packages/kbn-unified-data-table/src/utils/get_render_cell_popover.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiDataGridCellPopoverElementProps } from '@elastic/eui';
+import React, { memo, useEffect } from 'react';
+
+/*
+ *
+ * A custom cell popover render helps consumer of unified data table
+ * to get handle of unifiedDataTables's popover content and customize it.
+ *
+ * Default implementation is simply a pass through with just custom className.
+ * Consumers also have the ability to provide custom render functions
+ *
+ * */
+export const getCustomCellPopoverRenderer = () => {
+  return memo(function RenderCustomCellPopover(props: EuiDataGridCellPopoverElementProps) {
```

Comment:
More of a  stylistic nit than anything, but why not separate it out like so for readability?

```
const RenderCustomCellPopover = memo((props: props) => { ... });
export const getCustomCellPopoverRenderer = () => RenderCustomCellPopover;
````

Reply:
I would argue why do we need to keep the `RenderCustomCellPopover` name at all?
Wouldn't this be a  lot simpler?

```
export const getCustomCellPopoverRenderer = () =>
  memo((props: EuiDataGridCellPopoverElementProps) => {
    const { setCellPopoverProps, DefaultCellPopover } = props;
    {...}
    return <DefaultCellPopover {...props} />;
  });
```

Reply:
@PhilippeOberti  The problem is that it shows in `React Dev Tools` as Anonymous. It just a better DX to name the components.